datumaro/__init__.py

```diff
@@ -1,4 +1,3 @@
-
-# Copyright (C) 2019-2020 Intel Corporation
+# Copyright (C) 2019-2021 Intel Corporation
```

Comment:
I'm not sure it's valid to bump the copyright year of a file to which no changes have been made in 2021. Maybe you could revert changes to files that weren't modified this year?

Reply:
(Also, as an aside, having a copyright banner in this file at all is strange, since there's nothing else in it. Copyright law is pretty stringent, but I don't think _nothing_ is copyrighable. 🙂)